src/main/java/com/dumbdogdiner/stickyapi/common/arguments/Arguments.java

```diff
@@ -107,24 +107,35 @@ public Arguments requiredFlag(String name, String flag) {
         return this;
     }
 
-
     /**
-     * Create a required string argument.
+     * Create an optional string argument with a default value
      * @param name The name of this string
+     * @param fallback the default value you want for the argument
      * @return {@link com.dumbdogdiner.stickyapi.common.arguments.Arguments}
      */
-    public Arguments optionalString(String name) {
+    public Arguments optionalString(String name, String fallback){
         debug.print("Looking for optional string " + name + "...");
         if (unparsedArgs.size() > position) {
             parsedArgs.put(name, unparsedArgs.get(position));
             unparsedArgs.remove(position);
             debug.print("Found string at position " + String.valueOf(position) + " - new args size = " + String.valueOf(unparsedArgs.size()));
-        } else 
-            debug.print("Could not find string");
+        } else {
```

Comment:
Might be worth checking for a `null` fallback argument to avoid "falling back to `null`"

Reply:
The normal way it worked involved falling back to null, so I assume that's expected behavior. Do you have any suggestion on how best to fix this?

Reply:
When an optional arg isn't specified, it doesn't get added to the map at all - just in case somebody tries to use `containsKey` later - having a null value in the hashmap will still result in `HashMap.containsKey` returning true.

Reply:
Oh, you mean checking for null as the optional arg? I can do that

Reply:
I was trying to avoid duplicating code but I guess I can add another method